.azure-pipelines/common/package.yml

```diff
@@ -15,12 +15,72 @@ steps:
   displayName: 'Copy vsix to staging directory'
   inputs:
     Contents: '**/*.vsix'
-    TargetFolder: '$(build.artifactstagingdirectory)'
+    TargetFolder: '$(build.artifactstagingdirectory)/vsix'
```

Comment:
This is the work that Matt R. did to upload the web bundle to a storage account and output a SAS url.